devices/maas2/maas2.py

```diff
@@ -106,7 +110,7 @@ def _get_efi_data(self):
             "sudo efibootmgr -v",
         ]
         p = subprocess.run(
-            cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
+            cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, check=False
```

Comment:
Are all the check=False additions for linting purposes (I think it is the default) ?

Reply:
yes